HelpSource/Classes/Ndef.schelp

```diff
@@ -4,7 +4,9 @@ categories:: Libraries>JITLib>NodeProxy, Live Coding
 related:: Classes/ProxySpace, Classes/Tdef
 
 description::
-Reference to a proxy, forms an alternative to link::Classes/ProxySpace::. All methods are inherited from link::Classes/NodeProxy::.
+Ndef can be used to build a synth of which the internals can be replaced while running. An Ndef can be manipulated through its key. An Ndef can be started and stopped programmatically. The old synth and its replacement can automatically crossfade and the replacement time can be quantized. 
```

Comment:
Ndef, Pdef, and Tdef descriptions all use the word "programmatically," but I'm not sure what that means in this context. I think it would be more straightforward to merge two sentences and say "An Xdef can be started, stopped, and manipulated in other ways by referring to its key."

First sentence, I think you mean "a synth, the internals of which". Or it might be less awkward to phrase it as "a synth with internals that can be..."

Also, for Ndef, Pdef, Tdef, I think you should rephrase the first sentence in active voice (i.e. without "can be used") for directness: "Ndef builds...", "Pdef specifies...", "Tdef creates...".